styles/styles.css

```diff
@@ -485,8 +485,51 @@ main .section.underline-links a {
   text-decoration: underline;
 }
 
+/* viewport based visibility */
 
-/* section styles */
+@media screen and (max-width: 575px /* breakpoint-s */) {
+  main .hide-on-mobile {
+    display: none;
+  }
+}
+
+@media screen and (min-width: 576px /* breakpoint-s */) and (max-width: 1199px /* breakpoint-xl */) {
+  main .hide-on-tablet {
+    display: none;
+  }
+}
+
+@media screen and (min-width: 1200px /* breakpoint-xl */) {
+  main .hide-on-desktop {
+    display: none;
+  }
+}
+
+main .show-on-mobile,
+main .show-on-tablet,
+main .show-on-desktop {
+  display: none;
+}
+
+@media screen and (max-width: 575px /* breakpoint-s */) {
+  main .show-on-mobile {
+    display: unset;
```

Comment:
Maybe I'm wrong here. I thought it would reset to the default display style or one defined in other classes if I set it to `unset` here. `block` would certainly force this to be `block` ignoring other classes or whether the default is something else

Reply:
Defaulting to display: block is straightforward and allows easy overrides. Using display: unset can complicate maintenance and is generally unnecessary.

Reply:
Apart from this LGTM